cpp/include/cudf/detail/utilities/device_operators.cuh

```diff
@@ -29,6 +29,9 @@
 #include <cudf/utilities/error.hpp>
 #include <cudf/utilities/traits.hpp>
 
+#include <cuda/std/__algorithm/max.h>
+#include <cuda/std/__algorithm/min.h>
```

Comment:
The `min` and `max` functions are provided in either `cuda/std/functional` or `cuda/functional` (one of these should work, though I’m not certain which one). Relying on non-public headers, such as those with an underscore prefix like `__algorithm`, is not ideal.

Reply:
I agree that it's not ideal, but I wasn't able to get that to work when I tried it (based on a Slack thread I found where you asked similar questions). I'll try it again, and if it doesn't work I'll ping you offline.